man/systemd.network.xml

```diff
@@ -225,6 +225,14 @@
           if <literal>RequiredForOnline=no</literal>.</para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>Namespace=</varname></term>
+        <listitem>
+          <para>Takes a name or an abosolute file system path referring to a Linux network namespace
+          pseudo-file. If a name is provided, then it is assumed the file is in
+          <filename>/run/netns</filename>. If set, the interface is transferred to the namespace.</para>
```

Comment:
Do we want to use /run/netns here? From its documentation,  `ip netns` expects named network namespaces to reside in /var/run/netns. Not sure if we want to be compatible with that.

Reply:
I thought that `ip` command uses `/run/netns` as Fedora `/var/run` is a symlink to `/run`. Thanks.

Reply:
let's never mention /var/run/ ever. /run is the runtime dir, and if iproute2 wants to support 1990s linux systems that still have /var/run as a thing, then its on them, but we don't play that game with them...